pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -78,8 +78,13 @@ type Response struct {
 type connection struct {
 	fd *DeviceFD
 
+	// lock protect access to struct memebers.
+	lock sync.Mutex
```

Comment:
Please use "mu", see [style guide](https://github.com/google/gvisor/blob/master/g3doc/style.md). Also this is updated in the refactor PR #3548